Add HIP to support a release tag flag (#265)
helm/community: hips/hip-9999.md

---
hip: "9999"
title: "Helm Release Tag"
authors: [ "Nicholas Colbert <45cali@gmail.com>" ]
created: "2022-08-03"
type: "feature"
status: "draft"
---

## Abstract

> Add an optional release-tag flag for when deploying a helm chart.

Reviewer: ```suggestion
Add an optional release-tag flag that will be added to the release metadata.
```
